core/src/test/java/org/commonjava/maven/ext/manip/io/ModelResolverTest.java

```diff
@@ -52,9 +51,7 @@ public void resolveArtifactTest()
                             "cache-dir" ) );
         final GalleyAPIWrapper wrapper = new GalleyAPIWrapper( galleyInfra );
         final ModelIO model = new ModelIO();
-        final Field f = ModelIO.class.getDeclaredField( "galleyWrapper" );
-        f.setAccessible( true );
-        f.set(model, wrapper);
+        FieldUtils.writeField( model, "galleyWrapper", wrapper, true );
 
```

Comment:
Discussed offline with @jdcasey.

Adding a specific constructor means we also needed to create a default constructor for the plexus injection. I was trying to minimise accessors / improve encapsulation in the main code base. However reflection isn't _that_ nice either ;-) Alternatives might be to make the field package accessible and initialise it on creation or create a package accessible setter to use. And another solution is to use Mockitto `@InjectMocks` / `@Spy` which allows a real object to be injected in and keep the cleaner 'private' fields/ctors.